.github/dependency_tests.yaml

```diff
@@ -0,0 +1,12 @@
+broker[docker]: "tests/foreman/ -k 'test_host_registration_end_to_end or test_positive_erratum_applicability or test_positive_upload_content'"
+deepdiff: "tests/foreman/endtoend/test_api_endtoend.py -k 'test_positive_get_links'"
+dynaconf[vault]: "tests/foreman/api/test_ldapauthsource.py -k 'test_positive_endtoend'"
+manifester: "tests/foreman/api/test_subscription.py -k 'test_positive_create_after_refresh'"
+navmazing: "tests/foreman/ui/test_repository.py -k 'test_positive_create_as_non_admin_user'"
+pyotp: "tests/foreman/ui/test_ldap_authentication.py -k 'test_positive_login_user_password_otp'"
+pytest-xdist: "tests/foreman/ -n 3 -m 'build_sanity'"
+pytest: "tests/foreman/ -m 'build_sanity'"
+python-box: "tests/foreman/cli/test_webhook.py -k 'test_positive_end_to_end'"
+requests: "tests/foreman/cli/test_repository.py -k 'test_file_repo_contains_only_newer_file'"
+wait-for: "tests/foreman/api/test_reporttemplates.py -k 'test_positive_schedule_entitlements_report'"
+wrapanapi: "tests/foreman/longrun/test_provisioning_computeresource.py -k 'test_positive_provision_vmware_with_host_group'"
```

Comment:
VMWare tests usually fail due to unavailability of vlan_networking so would suggest better to use GCP or some other tests